packages/tabs/src/TabTitle/TabTitle.styles.ts

```diff
@@ -103,77 +177,50 @@ export const listTitleModeStyles: Record<Theme, ListTitleMode> = {
   },
 };
 
-export const listTitleFontSize: Record<BaseFontSize, string> = {
-  [BaseFontSize.Body1]: css`
+const sizeStyles: Record<'small' | 'default', string> = {
+  small: css`
+    padding: ${spacing[150]}px ${spacing[200]}px;
     font-size: ${typeScales.body1.fontSize}px;
```

Comment:
Are `font-size` and `line-height` necessary? It looks like they are already defined in `baseFontSizeStyles`

Reply:
the `'small'` size variant disregards the `baseFontSize` and always uses the smaller typography styles